modules/test/playwright/pages/portal-workflow-task-web/WorkflowTaskDetailsPage.ts

```diff
@@ -12,27 +12,53 @@ export class WorkflowTaskDetailsPage {
 	readonly approveMenuItem: Locator;
 	readonly assignee: FrameLocator;
 	readonly assignToMenuItem: Locator;
+	readonly subscribeButton: Locator;
 	readonly doneAssigneeButton: Locator;
 	readonly doneButton: Locator;
+	readonly commentBox: Locator;
+	readonly commentSectionButton: Locator;
 	readonly page: Page;
+	readonly rejectMenuItem: Locator;
+	readonly reply: Locator;
+	readonly viewButton: Locator;
 	readonly reviewActionMenu: Locator;
 	readonly reviewComment: Locator;
 	readonly workflowTasksPage: WorkflowTasksPage;
+	readonly detailsMessage: Locator;
+	readonly previewMessageBoards: Locator;
+	readonly activitiesButton: Locator;
+	readonly viewUsagesButton: Locator;
```

Comment:
Hey @rijema, could you sort these locators and their assignments in the constructor?

Reply:
It was missing ordering some of the items defined as readonly